editor/modes/visual-editor/index.js

```diff
@@ -40,6 +52,20 @@ class VisualEditor extends Component {
 		}
 	}
 
+	onKeyDown( event ) {
+		const { uids } = this.props;
+		const isEditable = [ 'textarea', 'input', 'select' ].indexOf( document.activeElement.tagName.toLowerCase() ) !== -1
+			|| document.activeElement.isContentEditable;
+		if (
+			! isEditable &&
+			( event.ctrlKey || event.metaKey ) &&
```

Comment:
On Windows, it's the Windows key (<kbd>⊞</kbd>), and conversely <kbd>⊞</kbd>+<kbd>a</kbd> is not a valid "select all" combination.